### PR TITLE
refactor: remove `Request` suffix in Msgs

### PR DESCRIPTION
**Pull request by atheeshp**
## Description

Closes: #399 

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] followed the guidelines for [building modules](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules)
- [ ] included the necessary unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed 
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

**Review by ruhatch — APPROVED** [review]
LGTM!

**Review by amaurym — APPROVED** [review]

**Comment by amaurym** [comment]
Tests still need to be fixed. We can safely ignore the `Protobuf / breakage` CI check here

**Review by anilCSE — APPROVED** [review]

**Comment by aaronc** [comment]
Has anyone looked into the test coverage issue here?

**Comment by atheeshp** [comment]
> Has anyone looked into the test coverage issue here?

I'm trying to fix this.

File: x/group/server/testsuite/genesis.go [hunk 1]
```diff
@@ -77,7 +78,14 @@ func (s *IntegrationTestSuite) TestInitExportGenesis() {
...
 
-	genesisData := map[string]json.RawMessage{group.ModuleName: genesisBytes}
+	ecocreditGenesisState := ecocredit.DefaultGenesisState()
+	ecocreditGenesisBytes, err := cdc.MarshalJSON(ecocreditGenesisState)
+	require.NoError(err)
+
+	genesisData := map[string]json.RawMessage{
+		group.ModuleName:     genesisBytes,
+		ecocredit.ModuleName: ecocreditGenesisBytes,
+	}
```
Comment (blushi) [inline x/group/server/testsuite/genesis.go:88]: I'm not so fond of having to put ecocredit module related logic in the group module tests but I guess we have no other choice with the current setup, unless we create public functions `InitGenesis` and `ExportGenesis` that could be tested individually without the server (they would be used in `serverImpl.InitGenesis` and `serverImpl.ExportGenesis`) similarly to what we have with the invariants tests. But then that would force us to set up all the tables again in the tests while this is already handled by the integration test suite here so this is not ideal either. So let's keep it as is for now until we come to a better solution in terms of module wiring / test setup.

**Review by blushi — APPROVED** [review]
Pre-approving, I left one comment that is good to keep in mind for the future but is out of scope of this PR.

**Pull request merged**